blockchain/validate.go

```diff
@@ -1161,7 +1163,11 @@ func (b *BlockChain) checkConnectBlock(node *blockNode, block *bchutil.Block, vi
 	// scripts.
 	transactions := block.Transactions()
 	totalSigOpCost := 0
-	maxSigOps := b.MaxBlockSigOps(uahfActive)
+	blockBytes, err := block.Bytes()
```

Comment:
I think you can use `block.MsgBlock().SerializeSize()` here

Reply:
That looks good! It just adds up the header size and then serialized size of all the transactions.

For my education, when we refer to block size, is that usually based on the wire serialization of the complete block? I saw that there is storage serialization (although it currently looks equivalent to wire), and also non-serialized views could have a size metric.

Reply:
yeah it should be the wire serialization.